dataland-frontend/src/components/wrapper/AuthenticationWrapper.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <slot v-if="authenticated"></slot>
+  <slot v-if="authenticated || disableAuthenticationWrapper"></slot>
```

Comment:
Ìs there a way in which this might be used to deactivate the authentication wrapper when it should not be possible?

Reply:
Of course there is a way, but it won't get you anything since you will only see a blank page if you are not really logged in, since you won't be able to fetch data without a keycloak token.

Reply:
ok